exercises/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,16 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return numbers[0];// complete this statement
 }
 
 function last(arr) {
-  return; // complete this statement
+ if (arr === numbers){
```

Comment:
This function passes the test, but uses "[magic numbers](https://www.codebyamir.com/blog/software-anti-patterns-magic-numbers)" to determine the last  element.

What property of an array can we use to determine the size or length of an array. Knowing the size or length of an array, how do we get the last element ( Hint, an arrays first element is at index zero, so the last is not the same as its length)